src/helper.h

```diff
@@ -31,6 +31,12 @@ struct get_json_ret_int {
   int v;
 };
 
+struct get_json_ret_array {
+  operator bool() const;
```

Comment:
This and other operators in this file should be marked `explicit` to avoid unintentional implicit conversions.